aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -29,23 +29,40 @@ This guide:
 
 ## Prerequisites
 
-1. Access to an Ubuntu 16.04 Server with a standard user account with sudo privilege
+1. Access to an Ubuntu 16.04 server with a standard user account with sudo privilege
 1. An existing ASP.NET Core app
 
 ## Copy over the app
 
-Run [dotnet publish](/dotnet/core/tools/dotnet-publish) from the dev environment to package an app into a self-contained directory that can run on the server.
+Configure the app for a [self-contained deployment](/dotnet/core/deploying/#self-contained-deployments-scd).
```

Comment:
I don't believe we want to actively guide customers to standalone due to concerns around servicing. Can we change this to be less opinionated?